index.html

```diff
@@ -41,10 +42,17 @@
 <body class='flex'>
   <div class="content">
     <div class="title">
-      <h1 class='h1'>Twin Cities Aid Map</h1>
+      <h1 data-translation-id="page_title" class='h1'>Twin Cities Aid Map</h1>
+      <ul class="languages">
+        <li><a data-lang="eng" href="/">English</a></li>
+        <li><a data-lang="spa" href="/?lang=spa">Español</a></li>
+        <li><a data-lang="kar" href="/?lang=kar">Karen</a></li>
+        <li><a data-lang="som" href="/?lang=som">Soomaali</a></li>
+        <li><a data-lang="hmn" href="/?lang=hmn">Hmong</a></li>
```

Comment:
It sounds like we might have folks that can do Amharic and Oromo too. Doesn't have to be added here. Can be a followup - just want you to know.

Reply:
@experimatt I am using https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes verses the more common ones like `en` or `en-US` to support languages like Hmong and Karen. I wasn't able to find standard 2-letter language codes for those.

Reply:
@andyinabox ahhhh gotcha, I didn't realize that. carry on then :)

Reply:
til another iso code